website/docs/content/writing-guidelines/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Writing guidelines
+caption: Consistent language guidelines for components and patterns.
+description: Consistent language guidelines for components and patterns.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+## Date and time
+
+Display dates and times clearly and consistently across products.
+
+### Usage
+
+These formatting options are available in the [Time](#) component. Choose the format most appropriate for your user needs.
```

Comment:
Todo: this can be done in the feature branch, but this link needs to be updated.